pages/portfolio.html

```diff
@@ -4,9 +4,75 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Projects Page</title>
+  <link rel="stylesheet" href="/style/style.css">
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <div id="logo"><a href="/pages/index.html" rel="index" class="logo">Home</a></div>
+      <ul>
+        <li><a href="/pages/portfolio.html" rel="portfolio">My Portfolio</a></li>
+        <li><a href="/pages/about.html" rel="about">About Me</a></li>
+        <li><a href="#social-footer" rel="contact">Contact Me</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="projects">
+      <h1>Projects Highlights</h1>
+      <div class="proj-container">
+        <div class="proj-grid">
+          <h3>
+            <a href="https://github.com/YLiu811/viewing-party.git">Viewing Party</a>
+          </h3>
+          <article>
+            <p>A Python program that replaces the spreadsheet method and allows users to keep track or movies watched and enjoyed.</p>
+            <p>Friends are able to share their favorite lists and watched lists. Friends can edit, delete, and add to watch list and favorite list.</p>
+          </article>
+          <img src="/imgs/viewing_party.png" alt="Viewing Party"/>
```

Comment:
It would be good for the alt text to be more descriptive. The picture itself is not of a party of people viewing something. It is a picture of VS Code displaying the code you wrote for this.